tests/test_crs/valid_vegeta_hostnetwork.yaml

```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: ripsaw.cloudbulldozer.io/v1alpha1
+kind: Benchmark
+metadata:
+  name: vegeta-benchmark-hostnetwork
+  namespace: my-ripsaw
+spec:
+  elasticsearch:
+    server: marquez.perf.lab.eng.rdu2.redhat.com
```

Comment:
Same as above - This needs to be updated to be ES_SERVER and ES_PORT for the elastic server/port with the new CI build. It passes those vars at runtime.